src/Http/Headers/src/HeaderUtilities.cs

```diff
@@ -625,34 +625,34 @@ public static StringSegment UnescapeAsQuotedString(StringSegment input)
         {
             input = RemoveQuotes(input);
 
-            // First pass to calculate the size of the InplaceStringBuilder
+            // First pass to calculate the size of the string
```

Comment:
Super unrelated but the formatting on line 625 seems to be off

Reply:
Looks fine in full file view: https://github.com/aspnet/AspNetCore/blob/51046dc8c1cb488169d42dcbee1b335b2157c663/src/Http/Headers/src/HeaderUtilities.cs#L625